src/components/Comments.jsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from "react"
+import fetchData from "../utils/api"
+import CommentCard from "./CommentCard"
+
+const Comments = ({articleId}) => {
+    const [comments, setComments] = useState([])
+
+    useEffect(() => {
+        fetchData({endpoint: `/articles/${articleId}/comments`})
+        .then(({comments}) => {
+            setComments(comments)
+        })
+    }, [])
+
+    return (
+        <div>
+            <h1>test</h1>
+            {comments.map((comment) => {
+            return <CommentCard key={comment.comment_id} comment={comment} />
+        })}
+        </div>
+    )
+}
```

Comment:
A note on the styling for this component, mobile looks great which is fab but desktop needs work as it uses the space weirdly:

![image](https://user-images.githubusercontent.com/61753159/192557962-70bd20d0-41a9-43e4-bfec-4805536ccda9.png)